### ISSUE TITLE
Missing primary keys

### ISSUE THREAD

**Issue by keyurshah** [putyourlightson/craft-blitz#237]
Hey Ben!

I'm using Craft Blitz on Digital Ocean and I'm getting an alert that they require primary keys on all their tables

I ran a diagnosis and it seems the following tables are missing a primary key

```
blitz_cachetags
blitz_elementcaches
blitz_elementexpirydates
blitz_elementquerycaches
blitz_elementquerysources
```

Can primary keys be added to these tables to avoid any future conflict with the database system. 

Thanks!

**Comment by bencroker** [comment]
Thanks Keyur, fixed in https://github.com/putyourlightson/craft-blitz/commit/01b7857e28179eb8e41548a50cd6fd677efe3963 for the next release.

For reference, this is something that Digital Ocean started enforcing [recently](https://www.digitalocean.com/docs/databases/#5-june-2020), as referenced by https://github.com/laravel/framework/issues/33238 and also solved by https://github.com/craftcms/cms/issues/6246.

**Comment by andrewfairlie** [comment]
Hi @bencroker thanks for this. Is there an ETA for this release, or a quick way to switch to this version pre-release?

**Comment by bencroker** [comment]
Most likely on Monday.

**Comment by bencroker** [comment]
Released in 3.6.6.
